vars/javaTemplate.groovy

```diff
@@ -103,6 +109,9 @@ def call(Map parameters = [:], body) {
         volumeMounts:
         - name: maven-settings
           mountPath: /root/.m2
+        resources:
+          requests:
+            memory: 2Gi
```

Comment:
What is the motivation behind this number? According to https://docs.gradle.org/current/userguide/gradle_daemon.html the default Heap memory is 512Mb so 1Gi should be sufficient.

Reply:
Our builds all go over 2Gi when built. 

1Gi will be fine since it's not going to be oomkilled since there is no limit. but if 3 builds run at the same time in QA its very likely to cause an issue.